TFT/src/User/Variants/Resolution/TFT_240X320.h

```diff
@@ -46,8 +46,8 @@
 #define STATUS_MSG_BODY_BOTTOM   4
 #define STATUS_GANTRY_YOFFSET    4
 
-#define NAME_LARGE_FONT          FONT_SIZE_NORMAL
-#define VAL_LARGE_FONT           FONT_SIZE_NORMAL
+#define SSICON_TITLE_FONT_SIZE   FONT_SIZE_NORMAL
```

Comment:
I know everyone has his style, but I would always define names using "_" to make them readable. In that sense I would use "SS_ICON_TITLE_FONT_SIZE" etc...

Reply:
sure